shared/test/integration/helpers/pageRegion.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var View = require('./view');
+
+function PageRegion(client, element) {
+	View.call(this, client);
+	this.waitForDisplay(element);
```

Comment:
Fixed. So basically the way I saw this implemented is that you chain findElement methods. Let me know if that's not correct.